### PR TITLE
MeetingRoom cmdlets - clarification for Teams

### PR DESCRIPTION
**Pull request by pbafna03**
*-CsMeetingRoom  cmdlets are not supported for managing Microsoft Teams Rooms but has been a cause of confusion, hence adding the note to clarify.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Disable-CsMeetingRoom.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ When you disable a meeting room object you remove all the Skype for Business Ser
 However, the Active Directory user account itself is not deleted.
 This cmdlet was introduced in Lync Server 2013.
 
+**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms, please use the methods mentioned in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation for the same.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Disable-CsMeetingRoom.md:22]:
```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

File: skype/skype-ps/skype/Enable-CsMeetingRoom.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ This cmdlet was introduced in Lync Server 2013.
 
 The process for creating and updating meeting rooms differs depending on your deployment of Skype for Business Online and Skype for Business Server.  Make sure you are following the guidance here to set up your meeting rooms properly: https://docs.microsoft.com/skypeforbusiness/deploy/deploy-clients/with-office-365
 
+**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms, please use the methods mentioned in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation for the same.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Enable-CsMeetingRoom.md:24]:
```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

File: skype/skype-ps/skype/Get-CsMeetingRoom.md [hunk 3]
```diff
@@ -18,6 +18,9 @@ A meeting room is a conferencing device designed to address video conferencing a
 This cmdlet was introduced in Lync Server 2013.
 
 
+**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms, please use the methods mentioned in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation for the same.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsMeetingRoom.md:21]:
```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

File: skype/skype-ps/skype/Set-CsMeetingRoom.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ Modifies the property values of an existing Skype for Business Server meeting ro
 A meeting room is a conferencing device designed to address video conferencing and collaboration scenarios in small conference rooms.
 This cmdlet was introduced in Lync Server 2013.
 
+**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms, please use the methods mentioned in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation for the same.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsMeetingRoom.md:20]:
```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pbafna03, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]
@pbafna03, thank you.

File: skype/skype-ps/skype/Disable-CsMeetingRoom.md [hunk 5]
```diff
@@ -19,6 +19,8 @@ When you disable a meeting room object you remove all the Skype for Business Ser
 However, the Active Directory user account itself is not deleted.
 This cmdlet was introduced in Lync Server 2013.
 
+**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```
Comment (dariomws) [inline skype/skype-ps/skype/Disable-CsMeetingRoom.md:22]:
```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. You must use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

Comment (dariomws) [reply skype/skype-ps/skype/Disable-CsMeetingRoom.md:22]: If they are not supported I think it's not a recommendation.

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsMeetingRoom.md [hunk 6]
```diff
@@ -18,6 +18,9 @@ A meeting room is a conferencing device designed to address video conferencing a
 This cmdlet was introduced in Lync Server 2013.
 
 
+**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsMeetingRoom.md:21]:
```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. You must use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

File: skype/skype-ps/skype/Set-CsMeetingRoom.md [hunk 7]
```diff
@@ -17,6 +17,8 @@ Modifies the property values of an existing Skype for Business Server meeting ro
 A meeting room is a conferencing device designed to address video conferencing and collaboration scenarios in small conference rooms.
 This cmdlet was introduced in Lync Server 2013.
 
+**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. We recommend you use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsMeetingRoom.md:20]:
```suggestion
**Note**: This cmdlet is not supported for managing Microsoft Teams Rooms. You must use the methods described in the [Microsoft Teams Rooms](/microsoftteams/rooms) documentation to manage Microsoft Teams Rooms.
```

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward please review

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**